crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -843,11 +843,17 @@ from this that attribute access on `Any` resolves to `Any` if the attribute does
 
 ```py
 from typing import Any
+from inspect import getattr_static
 
 class Foo(Any): ...
 
 reveal_type(Foo.bar)  # revealed: Any
-reveal_type(Foo.__repr__)  # revealed: Literal[__repr__] & Any
+
+reveal_type(getattr_static(Foo, "__repr__"))  # revealed: Literal[__repr__] & Any
+
+# TODO: This attribute access requires support for intersection types, since we try to access
+# `__get__` on `Literal[__repr__] & Any` (see above).
+reveal_type(Foo.__repr__)  # revealed: @Todo(Attribute access on `Intersection` types)
```

Comment:
Member access on intersection types is a pre-existing TODO that I did not solve in this PR.

Reply:
Ok, I added support for intersections now.